python/lsst/afw/geom/coordinates/coordinates.cc

```diff
@@ -364,6 +366,8 @@ PyPoint<T, N> declarePoint(py::module &mod, std::string const &suffix) {
     cls.def("__ne__", [](Point<T, N> const &self, Point<T, N> const &other) { return self != other; },
             py::is_operator());
 
+    cls.def("__hash__", [](Point<T, N> const &self) { return boost::hash<Point<T, N>>()(self); });
+
```

Comment:
I'm not sure it's a good idea to add this to Python; Points are not immutable and hence putting them in dicts could definitely confuse the dict.